MODULE.bazel

```diff
@@ -0,0 +1,69 @@
+module(name = "pico-sdk", version = "1.6.0-rc1")
+
+bazel_dep(name = "platforms", version = "0.0.8")
+
+# Note: rules_cc is special-cased repository; a dependency on rules_cc in a
+# module will not ensure that the root Bazel module has that same version of
+# rules_cc. For that reason, this primarily acts as a FYI. You'll still need
+# to explicitly list this dependency in your own project's MODULE.bazel file.
+bazel_dep(name = "rules_cc", version = "0.0.10")
+
+# rules_cc v0.0.10 is not yet cut, so manually pull in the desired version.
+# This does not apply to dependent projects, so it needs to be copied to your
+# project's MODULE.bazel too.
+archive_override(
+    module_name = "rules_cc",
+    urls = "https://github.com/bazelbuild/rules_cc/archive/71d9e52bdb292a9728ec8d18f46b33d596559e38.zip",
+    strip_prefix = "rules_cc-71d9e52bdb292a9728ec8d18f46b33d596559e38",
```

Comment:
I have an open commit that I need to include, so I'll be updating this shortly. If needed, I can get a full release of `rules_cc` cut before merging this PR.